dask/array/core.py

```diff
@@ -1835,6 +1839,14 @@ def clip(self, min=None, max=None):
         from .ufunc import clip
         return clip(self, min, max)
 
+    def divmod(self, other):
+        from .ufunc import divmod
+        return divmod(self, other)
+
+    def rdivmod(self, other):
+        from .ufunc import divmod
+        return divmod(other, self)
```

Comment:
It looks like these aren't standard. We can probably remove them.  This will be nice to keep the API slightly smaller.

Reply:
Agreed with @mrocklin , didn't see any other APIs with these type of function added along with them.

Reply:
It is standard. It just should be a dunder method.

ref: https://docs.python.org/3/reference/datamodel.html#object.__rdivmod__

Reply:
Right! I will do it

Reply:
Thanks for doing that 🙂